llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -429,6 +429,10 @@ class Triple {
   /// (SubArch).  This should only be called with Vulkan SPIR-V triples.
   VersionTuple getVulkanVersion() const;
 
+  /// Parse the DXIL version number from the OSVersion and DXIL version
+  /// (SubArch).  This should only be called with DXIL triples.
+  VersionTuple getDXILVersion() const;
```

Comment:
This comment doesn't seem accurate, given that `getDXILVersion` doesn't currently use the OS version

Reply:
Updated.